Test Program.sas

```diff
@@ -1 +1,6 @@
-/* Test Push */
\ No newline at end of file
+/* Test Push */
+/*I want to add my timer snippet, for seeing how long code takes*/
+
+options nonotes;data _null_;call symputx("starttime",time());run;options notes;
```

Comment:
I am a little confused why we need to have the _nonotes_ options here?

Reply:
We need it so that we can turn off notes for the `symputx` code call, and then turn them back on in case notes are needed for the code chunk that we are timing.